homeassistant/components/system_bridge/sensor.py

```diff
@@ -240,7 +245,7 @@ async def async_setup_entry(
             SystemBridgeSensor(coordinator, description, entry.data[CONF_PORT])
         )
 
-    for key, _ in coordinator.data["disk"].items():
+    for key, _ in coordinator.data.disk.__dict__.items():
```

Comment:
If we need a dict from the pydantic model we can use the `.dict` method on the model.

https://pydantic-docs.helpmanual.io/usage/models/#basic-model-usage